enforcer-rules/README.md

```diff
@@ -1,117 +1,7 @@
 # Linkage Checker Enforcer Rule
 
-This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
```

Comment:
wouldn't hurt to keep this first paragraph here to explain what it does

Reply:
That's nice idea. Updated.